pkg/apis/sparkoperator.k8s.io/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -407,16 +407,6 @@ func (in *SparkApplicationList) DeepCopyObject() runtime.Object {
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *SparkApplicationSpec) DeepCopyInto(out *SparkApplicationSpec) {
 	*out = *in
-	if in.Image != nil {
-		in, out := &in.Image, &out.Image
-		*out = new(string)
-		**out = **in
-	}
```

Comment:
Not sure if these are needed, since it is no longer string

Reply:
These are automatically generated so we just take whatever changes the code-gen made.

Reply:
Run `hack/update-codegen.sh` and `hack/verify-codegen.sh`.